src/arkit.cs

```diff
@@ -2034,8 +2051,12 @@ interface ARSkeletonDefinition {
 		[NullAllowed, Export ("neutralBodySkeleton3D")]
 		ARSkeleton3D NeutralBodySkeleton3D { get; }
 
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
 		[Export ("indexForJointName:")]
-		nuint GetJointIndex ([BindAs (typeof (ARSkeletonJointName))] NSString jointName);
+		nuint GetJointIndex (NSString? jointName);
+
+		[Wrap ("GetJointIndex (jointName.GetConstant()!)")]
+		nuint GetJointIndex (ARSkeletonJointName jointName);
```

Comment:
Because the new added C API allows to create new ARSkeletonJointName which are a typedef based on NSString, if we hide it then the new API makes so sense. The new binding exposes the NSString option as an advance option and then adds the wrap for our smart enum.

Reply:
make sense, copy/paste that into the commit message before merging :)